out.txt

```diff
@@ -0,0 +1,7274 @@
+[  4%] Built target threadpool
```

Comment:
I don't think this file should have been checked in ;-)

Reply:
Removed

Reply:
Sorry about resolving.
They are now unresolved

Reply:
No biggie. They were, after all, correctly resolved ;-)